app/move/controllers/view/view.locals.js

```diff
@@ -16,16 +16,6 @@ function getViewLocals(req) {
     cancellation_reason_comment: cancellationComments,
     rejection_reason: rejectionReason,
   } = move
-
-  // We have to pretend that 'secure_childrens_home', 'secure_training_centre' are valid `move_type`s
```

Comment:
Do we no longer need to pretend that these are valid move types?

Reply:
It's in the Nunjucks template now

Reply:
Yeah, what @solidgoldpig said. 

It was introduced in #757 and looked like it was to cater for variations to the [pending review content](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/a7efddb5e67a08870ebbcfd7fc258d4a9200f366/locales/en/messages.json#L17-L18).

After digging into it, there were no other variations based on `move_type`, so instead, it now passes the `from_location.location_type` in as context which allows the same overrides to be set.

It essentially felt more dangerous to be overriding the value from the API to update content, instead it feels clearer to pass in the context we want to control content based on.

Reply:
Ok, sounds good.